WorkflowUI/Sources/Container/ContainerViewController.swift

```diff
@@ -69,8 +75,10 @@
             fatalError("init(coder:) has not been implemented")
         }
 
-        private func render(screen: ScreenType, environment: ViewEnvironment) {
-            rootViewController.update(screen: screen, environment: environment)
+        private func update(screen: ScreenType, environment: ViewEnvironment) {
+            update(child: \.rootViewController, with: screen, in: environment)
+
+            updatePreferredContentSizeIfNeeded()
```

Comment:
Nice

Reply:
If we're swapping out the inner view controller impl, we need to re-ask for its size. The fact we don't do this in `DescribedViewController` seems like a bug; I can fix here.

- [ ] Fix `DescribedViewController`